.github/workflows/cypress-tests.yml

```diff
@@ -6,8 +6,6 @@ name: Cypress Tests
 on:
   push:
     branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
 
 jobs:
   cypress-run:
```

Comment:
```suggestion
  cypress-run:
    runs-on: ubuntu-latest
    container:
      image: cypress/browsers:latest
```

Reply:
This might work. Also it might be worth testing this in `cypress-tests` before merging into `main`

Reply:
Seems like this solved the missing `Xvfb` when running the GitHub action with `Act`. Though https://github.com/cypress-io/github-action does not mention using the image